concourse/setup.py

```diff
@@ -257,6 +260,49 @@ def create_instance_specific_helm_values(
     return instance_specific_values
 
 
+def create_webhook_dispatcher_helm_values(
+    webhook_dispatcher_cfg: WebhookDispatcher,
+    cfg_factory: ConfigFactory,
+):
+    not_none(webhook_dispatcher_cfg)
+    not_none(cfg_factory)
+
+    # create secrets server endpoint
+    secrets_server_name = webhook_dispatcher_cfg.secrets_server()
+    secrets_server_cfg = cfg_factory.secrets_server(secrets_server_name)
+    secrets_server_endpoint = '/'.join([
+        secrets_server_cfg.endpoint_url(),
+        secrets_server_cfg.secrets().concourse_secret_name(),
+        secrets_server_cfg.secrets().concourse_attribute()])
+
+    concourse_cfg_names = webhook_dispatcher_cfg.concourse_cfgs()
+    github_cfg_names = webhook_dispatcher_cfg.github_cfgs()
+    job_mapping_cfg_names = webhook_dispatcher_cfg.job_mappings()
+
+    env_vars = []
+    env_vars.append({
+        'name': 'SECRETS_SERVER_ENDPOINT', 'value': secrets_server_endpoint
```

Comment:
why? just pass the one relevant cfg_name (which would be the secrets-server cfg name)
also (but irrelevant): why not use a DICT?

Reply:
Done